### DISCUSSION TITLE
Feature request: ZFS Smart Status

### DISCUSSION THREAD

**Discussion by cryd-s** [dougiteixeira/proxmoxve#114] [Feature request] [answered]
It would be a pleasure to have the health of the zfs storages and the disk health.

**Comment by caublet** [comment]
would love this indeed

**Comment by dougiteixeira** [comment, marked as answer]
It has been added and is available in [version 2.2.0](https://github.com/dougiteixeira/proxmoxve/releases/tag/2.2.0).

Please if you can test and give me feedback if everything works well.
